### ISSUE TITLE
Test creating a new profile from scratch

### ISSUE THREAD

**Issue by jgreben** [LD4P/sinopia_profile_editor#50]
- Visit http://localhost:8000/#/profile/create
- Click Add Resource Template button: `#addResource`
- Expect:
```
<span popover="Date: Oct 03 2018" popover-title="Profile ID: Undefined" popover-placement="right" popover-trigger="mouseenter" class="ng-scope">Profile</span>
``` 
- Expect:
```
<span popover="URI: Undefined" popover-title="Resource ID: Undefined" popover-trigger="mouseenter" popover-placement="right" class="ng-scope ng-binding">Resource Template
</span>
```
- Expect <div> with selector: `#resource_{n}` (some number)
- Expect `<table>` with 5 `<input>`
- Click `Add Property Template` button with `ng-click="addProperty()"`
- Expect page to have 5 select elements and (now) 16 input elements
- Click Add Resource Template button again: `#addResource`
- Expect now 2 elements:
```
<span popover="URI: Undefined" popover-title="Resource ID: Undefined" popover-trigger="mouseenter" popover-placement="right" class="ng-scope ng-binding">Resource Template
</span>
```
- Expect `<div>` with selector: `#resource_{n+1}` (some number incremented from above)
- Click first selector #deleteButton with `ng-click="verifyDelete(deleteResource)"`
- Expect button with `ng-click="confirm();"` and click button
- Click first selector #deleteButton with `ng-click="verifyDelete(deleteResource)"`
- Expect page not to have `<div>` with selector: `#resource_{n}` (some number)

**Comment by jgreben** [comment]
We could even add more expectations for adding Templates and sub-resources to the resource template, but maybe that would be overkill(?).

**Comment by ndushay** [comment]
Is #54 a dup?  if so, please close it.

**Comment by jgreben** [comment]
Fixed by #68
